executor/licenses/dep.txt

```diff
@@ -228,7 +228,6 @@ github.com/jstemmer/go-junit-report
 github.com/jtolds/gls
 github.com/julienschmidt/httprouter
 github.com/jung-kurt/gofpdf
-github.com/kedacore/keda
```

Comment:
Is this from a missed license run from a different PR?

Reply:
It seems that way. Either that, or some subdep got updated. The linter failed within the GH Action, and re-generating the licenses file locally fixed it.